ifrs17/Test/TechnicalMarginTest.ipynb

```diff
@@ -364,10 +506,278 @@
+                "\nvar inputDataSet = new IfrsVariable[]{",
+                "\n    grossBasicIfrsVariable with {AocType = \"BOP\", Novelty = \"N\", Value = 100.0},",
+                "\n    grossBasicIfrsVariable with {AocType = \"IA\", Novelty = \"N\", Value = -50.0}, ",
+                "\n    grossBasicIfrsVariable with {AocType = \"AM\", Novelty = \"C\", Value = 0.5, EstimateType = \"F\", AmountType = null},",
+                "\n",
+                "\n    reinsBasicIfrsVariable with {AocType = \"BOP\", Novelty = \"N\", Value = -100.0},",
+                "\n    reinsBasicIfrsVariable with {AocType = \"IA\", Novelty = \"N\", Value = -10.0},",
+                "\n    reinsBasicIfrsVariable with {AocType = \"CL\", Novelty = \"C\", Value = -30.0},",
+                "\n    reinsBasicIfrsVariable with {AocType = \"AM\", Novelty = \"C\", Value = 0.5, EstimateType = \"F\", AmountType = null},",
+                "\n  };",
+                "\n",
+                "\n//Gross CSM-LC",
+                "\nvar csmLcSwitch_benchmark = new Dictionary<AocStep,(double valueCsm, double valueLc)>()",
+                "\n    {",
+                "\n        {new AocStep(\"BOP\",\"N\"), (0d, 100d)},",
+                "\n        {new AocStep(\"IA\",\"N\"), (0d,  -50d)}, //IA of the gross has opposite sign wrt the others (triggers switching)",
+                "\n        {new AocStep(\"AM\",\"C\"), (0d,  -25d)},",
+                "\n        {new AocStep(\"EOP\",\"C\"),(0d,   25d)},",
+                "\n    };",
+                "\n",
+                "\nvar reinsCsmLcSwitch_benchmark = new Dictionary<AocStep,(double valueCsm, double valueLc)>()",
+                "\n    {",
+                "\n        {new AocStep(\"BOP\",\"N\"),(0d,   -100d)},",
+                "\n        {new AocStep(\"IA\",\"N\"),(0d,     -10d)},",
+                "\n        {new AocStep(\"CL\",\"C\"), (0d,    -30d)},",
+                "\n        {new AocStep(\"AM\",\"C\"), (0d,     70d)},",
+                "\n        {new AocStep(\"EOP\",\"C\"),(0d,    -70d)},",
+                "\n    };"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "var activity = await CheckSwitchLogicAsync(inputDataSet, csmLcSwitch_benchmark, reinsCsmLcSwitch_benchmark);",
+                "\nactivity"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "activity.Status.Should().Be(ActivityLogStatus.Succeeded);"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## Reinsurance with Gross with CL balancing item"
```

Comment:
Should it be "LC balancing item" instead?

Reply:
it's actually CL: combined liability AoC Step